.github/workflows/securityAlertsReview.yml

```diff
@@ -84,10 +84,16 @@ jobs:
           ALERTS=$(curl -s -H "Authorization: token ${GITHUB_TOKEN}" \
             "https://api.github.com/repos/${{ github.repository }}/code-scanning/alerts?pr=${PR_NUMBER}")
 
-          # Log raw API response for debugging
           echo "Raw API Response:"
           echo "$ALERTS"
 
+          echo "Filtering to Olympix Static Analysis alerts only"
+          ALERTS=$(echo "$ALERTS" | jq -c '[ .[] | select(.tool.name == "Olympix Integrated Security") ]')
+
+          # Log raw API response for debugging
```

Comment:
nit: change to `filtered` instead of `raw`?

Reply:
updated: deleted this comment completely as below there is a proper echo message